### PR TITLE
correct some comments

### PR DESCRIPTION
**Pull request by B1F030**
#### What type of PR is this?
/kind document

#### What this PR does / why we need it:
correct some comment spelling and grammar

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

**Review by mimowo — COMMENTED** [review]
/lgtm
Thank you!
/assign @tenzen-y @alculquicondor

File: pkg/util/heap/heap_test.go [hunk 1]
```diff
@@ -136,8 +136,8 @@ func TestHeap_PushOrUpdate(t *testing.T) {
 	h := New(testHeapObjectKeyFunc, compareInts)
 	h.PushOrUpdate(mkHeapObj("foo", 100))
 	h.PushOrUpdate(mkHeapObj("baz", 20))
-	h.PushOrUpdate(mkHeapObj("foo", 1)) // This behaviors as update.
-	h.PushOrUpdate(mkHeapObj("zab", 8)) // This behaviors as add.
+	h.PushOrUpdate(mkHeapObj("foo", 1)) // This manifests as update.
+	h.PushOrUpdate(mkHeapObj("zab", 8)) // This manifests as add.
```
Comment (tenzen-y) [inline pkg/util/heap/heap_test.go:140]: I think that previous comments are correct.

File: pkg/scheduler/scheduler.go [hunk 2]
```diff
@@ -158,7 +158,7 @@ func (s *Scheduler) schedule(ctx context.Context) {
 	// 1. Get the heads from the queues, including their desired clusterQueue.
 	// This operation blocks while the queues are empty.
 	headWorkloads := s.queues.Heads(ctx)
-	// No elements means the program is finishing.
+	// No elements, means the program is finishing.
```
Comment (tenzen-y) [inline pkg/scheduler/scheduler.go:161]:
```suggestion
	// No elements means the program is finishing.
```

File: pkg/features/kube_features.go [hunk 3]
```diff
@@ -78,7 +78,7 @@ func SetFeatureGateDuringTest(tb testing.TB, f featuregate.Feature, value bool)
 	return featuregatetesting.SetFeatureGateDuringTest(tb, utilfeature.DefaultFeatureGate, f, value)
 }
 
-// Helper for `utilfeature.DefaultFeatureGate.Enabled()`
+// Enabled helper for `utilfeature.DefaultFeatureGate.Enabled()`
```
Comment (tenzen-y) [inline pkg/features/kube_features.go:81]:
```suggestion
// Enabled is helper for `utilfeature.DefaultFeatureGate.Enabled()`
```

File: pkg/util/heap/heap_test.go (hunk 1 above)
Comment (B1F030) [reply pkg/util/heap/heap_test.go:140]: behavior is a noun, i think "behavior as" can be replaced with "manifest as", "represent as" or "behave as"

File: pkg/scheduler/scheduler.go (hunk 2 above)
Comment (B1F030) [reply pkg/scheduler/scheduler.go]: How about this:
If there are no elements, it means that the program is finishing.

Comment (tenzen-y) [reply pkg/scheduler/scheduler.go]: SGTM

File: pkg/util/heap/heap_test.go (hunk 1 above)
Comment (tenzen-y) [reply pkg/util/heap/heap_test.go:140]: I like "behave as" since `PushOrUpdate` is a behavior.

**Review by tenzen-y — COMMENTED** [review]
Thanks!
/lgtm
/approve

**Pull request merged**